### DISCUSSION TITLE
plain text body?

### DISCUSSION THREAD

**Discussion by hyusetiawan** [rootstrap/django-drip-campaigns#186] [General]
I see in the admin that we can define the HTML body, but where can we define the text version? Having a text version increases the deliverability of the email.

**Comment by brunomichetti** [comment]
> I see in the admin that we can define the HTML body, but where can we define the text version? Having a text version increases the deliverability of the email.

Hello @hyusetiawan sorry for the late response. There was no one reviewing these questions. We are trying to continue the maintenance of the project and our team will address this question as soon as we can.
